projectOne/src/oneFive.java

```diff
@@ -0,0 +1,85 @@
+import java.util.Arrays;
+import java.util.Scanner;
+public class oneFive {
+    public static void main(String[] args) {
+            Scanner scanner = new Scanner(System.in);
+            String name = "константин";
+            String lastName = "прадед";
+            String nameArray[] = {"к","о","н","с","т","а","н","т","и","н"};
+            String nameArrayEmpty[] = {"*","*","*","*","*","*","*","*","*","*"};
+            String lastNameArray[] = {"п","р","а","д","е","д"};
+            String lastNameEmptyArray[] = {"*","*","*","*","*","*"};
+            String usedLettersArray [] = new String [100];
+            int nextIndex = 0;
+        while (!(Arrays.equals(nameArray,nameArrayEmpty) && Arrays.equals(lastNameArray,lastNameEmptyArray))) {
+            System.out.println("Введите слово или букву чтобы угадать имя и фамилию");
+            String input = scanner.nextLine();
```

Comment:
you call toLowerCase() almost everywhere where you call this variable, I think it will be better to keep it already in lower case